opentreemap/treemap/instance.py

```diff
@@ -526,21 +535,26 @@ def add_map_feature_types(self, types):
 
         add_default_permissions(self, models=classes)
 
+    def update_all_revs(self):
+        for rev in ('geo_rev', 'eco_rev', 'universal_rev'):
+            self._update_rev(rev)
```

Comment:
Do we need to be concerned about making 3 SQL `UPDATE` queries one after another? We could write an `_update_revs` that takes a list of attributes and makes a single `qs.update` call.